multibase/main.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"fmt"
+	"github.com/multiformats/go-multibase"
+	"io/ioutil"
+	"os"
+
+	"github.com/urfave/cli/v2"
+)
+
+func main() {
+	app := &cli.App{
+		Name:  "multibase",
+		Usage: "base encoding and transcoding tool",
+		Commands: []*cli.Command{
+			{
+				Name:      "encode",
+				ArgsUsage: "<base>",
```

Comment:
```suggestion
				ArgsUsage: "<base> [<data>]",
```

Reply:
Also, maybe make the base a flag? I.e., `--base=xyz`? That way, we can set a reasonable default base (e.g., base32) so users can just call `$(multibase encode "foobar")` and get the result they expect.